README.md

```diff
@@ -112,21 +112,25 @@ working on the code:
 
 ### Steps
 
-1.  Install [Air](https://github.com/cosmtrek/air#installation)
+1. Install [Air](https://github.com/cosmtrek/air#installation)
 
-1.  Pull source
+2. Pull source
```

Comment:
nit: I personally prefer to always use 1. so that we don't need to change the number if we insert a step in between. wdyt?

Reply:
It seems like GoLand automatically update them for me 🤦

Reply:
Fixed.